dbaas-lib/engines/k8s-psmdb/types/v110/psmdb_types.go

```diff
@@ -85,6 +85,17 @@ func (cr *PerconaServerMongoDB) Upgrade(imgs map[string]string) {
 	}
 }
 
+func (cr *PerconaServerMongoDB) GetStatus() string {
+	return string(cr.Status.Status)
```

Comment:
I think it's better to return here not a string but typed dbaas status. So, later on, you can compare it without casting to the string.

Reply:
https://github.com/Percona-Lab/percona-dbaas-cli/pull/101/commits/0dccdc08eeca4fe39a3cbf9e7e558227fbe775c5